vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -335,6 +336,24 @@ public void testFilterUsingIntersectingRegions() throws Exception {
         checkOrderInOutputFile(outputFile);
     }
 
+    @Test
+    public void testDivideChromosomeInChunks() throws Exception {
+        String studyId = "7";
+        List<String> studies = Collections.singletonList(studyId);
+        MultivaluedMap<String, String> queryParameters = new MultivaluedHashMap<>();
+        int blockSize = Integer.parseInt(evaTestProperties.getProperty("eva.htsget.blocksize"));
+        VariantExporterController controller = new VariantExporterController(
+                TestDBRule.getTemporaryDBName(TestDBRule.HUMAN_TEST_DB),
+                studies, evaTestProperties, queryParameters, blockSize);
+        List<Region> regions = controller.divideChromosomeInChunks("1", 500, 1499);
+        assertTrue(regions.size() == 1);
+        assertTrue(regions.contains(new Region("1", 500, 1499)));
+
+        regions = controller.divideChromosomeInChunks("1", 500, 1000);
```

Comment:
Could we make the number of regions different for this check? Otherwise it will be mostly a repetition of the previous one.